cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -440,6 +463,28 @@ CREATE INDEX ...
 a|
 The default index type is changed from B-tree to range index.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW INDEXES
+----
+a|
+A new column is added, `owningConstraint`. The column is returned by default.
+This column returns the name of the constraint the index is associated with, or `null` if it is not associated with any constraint.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW CONSTRAINTS
+----
+a|
+A new column is added, `ownedIndex`. The column is returned by default.
+This column returns the name of the index associated with the constraint, or `null` if no index is associated with the constraint.
```

Comment:
```suggestion
It will list the name of the index associated with a constraint or `null`, in case no index is associated with it.
```

Reply:
```suggestion
It will list the name of the index associated with the constraint or `null`, in case no index is associated with it.
```
you kept the `the` for the index equivalent sentence (which I think sounds better)

Reply:
Agreed!